views/index.ejs

```diff
@@ -70,11 +70,14 @@
                                     <% } %>
                                 </div>
                                 <div class="col-4 text-end">
-                                    <div class="icon icon-shape bg-gradient-primary shadow text-center border-radius-md">
-                                        <% if (stats.icon.startsWith('fa-')) { %>
-                                            <i class="<%= stats.icon %> text-lg opacity-10" aria-hidden="true"></i>
-                                        <% } else { %>
-                                            <i class="ni ni-<%= stats.icon %> text-lg opacity-10" aria-hidden="true"></i>
+                                    <div class="icon icon-shape text-center border-radius-md"
```

Comment:
might wanna readd "bg-gradient-primary"

Reply:
There should be no background when using a custom image

Reply:
okay coolio great work than @X-AmpleDevelopment :)

Reply:
Thanks both!